src/storage/mvcc/mod.rs

```diff
@@ -63,10 +63,10 @@ quick_error! {
             description("txn lock not found")
             display("txn lock not found {}-{} key:{:?}", start_ts, commit_ts, escape(key))
         }
-        WriteConflict { start_ts: u64, conflict_ts: u64, key: Vec<u8>, primary: Vec<u8> } {
+        WriteConflict { start_ts: u64, conflict_start_ts: u64, conflict_commit_ts: u64, key: Vec<u8>, primary: Vec<u8> } {
             description("write conflict")
-            display("write conflict {} with {}, key:{:?}, primary:{:?}",
-             start_ts, conflict_ts, escape(key), escape(primary))
+            display("write conflict {} with {},{}, key:{:?}, primary:{:?}",
```

Comment:
please make the words more concrete, or we can't know what these values stands for directly from log.